src/otx/core/utils/mask_util.py

```diff
@@ -27,4 +33,43 @@ def polygon_to_bitmap(polygons: list[Polygon], height: int, width: int) -> np.nd
     """
     polygons = [polygon.points for polygon in polygons]
     rles = mask_utils.frPyObjects(polygons, height, width)
+    if return_rle:
+        return rles
     return mask_utils.decode(rles).astype(bool).transpose((2, 0, 1))
+
+
+def encode_rle(mask: torch.Tensor) -> dict:
```

Comment:
Would you please add an unit test to validate whether this function is equivalent to `pycocotools.encode()`?

Reply:
Added in `tests/unit/core/utils/test_mask_utils.py`